pkgs/development/libraries/java/httpunit/default.nix

```
@@ -1,15 +1,23 @@
-{lib, stdenv, fetchurl, unzip} :
+{ lib, stdenv, fetchurl, unzip }:
 
-stdenv.mkDerivation {
-  name = "httpunit-1.7";
-  builder = ./builder.sh;
+stdenv.mkDerivation rec {
+  pname = "httpunit";
+  version = "1.7";
 
   src = fetchurl {
-    url = "mirror://sourceforge/httpunit/httpunit-1.7.zip";
+    url = "mirror://sourceforge/httpunit/httpunit-${version}.zip";
     sha256 = "09gnayqgizd8cjqayvdpkxrc69ipyxawc96aznfrgdhdiwv8l5zf";
   };
 
-  inherit unzip;
+  nativeBuildInputs = [ unzip ];
+
+  installPhase = ''
```

By this point, unzip should be already executed in the `unpackPhase`. I suggest to either use the unpacked copy or to change this to
```suggestion
  buildCommand = ''
```
(you don't need hooks with it)

---

thats good to know. thanks